app/models/news.py

```
@@ -20,3 +21,8 @@ class Article(BaseModel):
 class NewsResponse(BaseModel):
     totalArticles: int
     articles: List[Article]
+
+
+class SingleNewsArticleResponse(BaseModel):
```

Pylint (3.3.7)

[convention] Missing class docstring

(C0115)